File: Java/Factorial.java
import java.util.Scanner;

public class Factorial {
     public static void main(String[] args) {
          Scanner sc = new Scanner(System.in);
          System.out.print("Enter the number: ");
          int num = sc.nextInt();
          int factorial = 1;

          for (int i = 1; i <= num; i++) {
               factorial = factorial * i;
          }

          System.out.println("The factorial of " + num + " is " + factorial);

          sc.close();
     }
}

File: Python/factorial.py
# n!= 1*2*3*4*5*.....*n
# 5!= 1*2*3*4*5 = 120

num = int(input("Enter the number:\n"))
factorial = 1
for i in range(1, num + 1):
    factorial *= i
print("The factorial of", num, "is", factorial)
